### DISCUSSION TITLE
BeatmapInfoWedge mapper doesn't reflect guest difficulty one, but opens their profile upon click

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#23233] [Q&A] [answered]
This isn't relevant to the new release, happened before too.
Now that i think about it, it also isn't displayed in the carousel next to the diff name. I think it should be changed, but i don't know if it's even feasible.
(i searched before posting disclaimer)

https://user-images.githubusercontent.com/53081750/233106392-39710da8-48a3-4e8e-b0d4-66c69bb3e418.mp4

**Comment by Hifumindesu** [comment, marked as answer]
Should be fixed once the new song select is fully implemented (#18890)

The new design displays the name of the mapset host/creator separate from the difficulty mapper

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/issues/12965
